cl/simple_pages/templates/help/alert_help.html

```diff
@@ -238,7 +238,7 @@ <h3>Creating Alerts</h3>
     </p>
 
     <h3>Enabling Real Time Alerts</h3>
-    <p>Real Time alerts are only available for supporters of Free Law Project that have donated at least ten dollars over the past year. If you see a message like the one below, you will <a href="{% url "donate" %}?referrer=alerts-help">need to donate</a> to enable Real Time alerts:
+    <p>Real Time alerts are only available for supporters of Free Law Project that have donated at least ten dollars over the past year. If you see a message like the one below, you will <a href="https://donate.free.law/forms/membership">need to donate</a> to enable Real Time alerts:
```

Comment:
This pricing is no longer correct. Now they have to be monthly or legacy donors. So:

```suggestion
    <p>Real Time alerts are only available for supporters of Free Law Project. If you see a message like the one below, you will <a href="https://donate.free.law/forms/membership">need to become a member</a> to enable Real Time alerts:
```